src/back/TicketManager/tests/Core/TicketManager.Core.ServicesTests/Endpoints/Roles/RoleTests/AccountRoleTests.cs

```diff
@@ -0,0 +1,21 @@
+using Xunit;
+using TicketManager.Core.Services.Endpoints.Accounts;
+using TicketManager.Core.Services.Services.TokenManager;
+
+namespace TicketManager.Core.ServicesTests.Endpoints.Roles.RoleTests;
+
+public class AccountRoleTests
+{
+    private readonly RoleTestsBase testsBase;
+    public AccountRoleTests()
+    {
+        testsBase = new RoleTestsBase();
+    }
+
+    [Fact]
+    public void WhenAccountLoginChecked_ItShouldAllowAnonymous()
+    {
+        var testInstance = testsBase.GetRoleTestInstance<AccountLoginEndpoint>();
+        testInstance.AsAnonymous();
```

Comment:
I would think about name of `AsAnonymous` method. It is named the same way as other methods in `RoleTestInstance` class, but how we use it is a little bit different. Or I would check if `Check` method works here, because I guess it does.

Reply:
`Check` doesn't really work here because it checks if `List` is empty, but I can rename it to something like `CheckAnonymous`